html/semantics/embedded-content/the-img-element/image-loading-lazy-crossorigin-change.sub.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<head>
+  <title>Deferred images with loading='lazy' use the current crossorigin attribute</title>
+  <link rel="author" title="Raj T" href="mailto:rajendrant@chromium.org">
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="common.js"></script>
+</head>
+
+<script>
+  const crossorigin_img = new ElementLoadPromise("crossorigin_img");
+
+  // Set the crossorigin and scroll down to load the deferred image.
+  window.addEventListener("load", () => {
+    crossorigin_img.element().crossOrigin = 'anonymous';
+    crossorigin_img.element().scrollIntoView();
+  });
+
+  async_test(function(t) {
+    crossorigin_img.promise.then(t.unreached_func("The image should not load.")).catch(t.step_func_done());
+  }, "Test that when deferred image is loaded, it uses the current crossorigin attribute.");
+</script>
+
+<body>
+  <div style="height:10000px;"></div>
```

Comment:
Could we make this `vh` instead of `px`, kind of like what we did in some of the other tests?